doc/source/release-notes/Percona-Server-5.7.10-2.rst

```diff
@@ -0,0 +1,80 @@
+
+Based on `MySQL 5.7.10 <http://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-10.html>`_, including all the bug fixes in it, |Percona Server| 5.7.10-2 is the current Release Candidate release in the |Percona Server| 5.7 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.7.10-2 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.7.10-2rc2>`_
+
+New Features
+============
+
+ Complete list of changes between |Percona Server| 5.6 and 5.7 can be seen in :ref:`changed_in_57`.
+
+ 5.7 binlog group commit algorithm is now supported in |TokuDB| as well.  
+
+Known Issues
+============
+
+ In |Percona Server| 5.7 `super_read_only <https://www.percona.com/doc/percona-server/5.6/management/super_read_only.html>`_ feature has been replaced with upstream implementation. There are currently two know issues compared to |Percona Server| 5.6 implementation:
+
+   * Bug :mysqlbug:`78963`, :variable:`super_read_only` aborts ``STOP SLAVE`` if variable :variable:`relay_log_info_repository` is set to ``TABLE`` which could lead to a server crash in Debug builds.
+
+   * Bug :mysqlbug:`79328`, :variable:`super_read_only` set as a server option has no effect.
+
+ Restarting the server with the :variable:`innodb_flush_method` set to ``ALL_O_DIRECT`` fails intermittently which could break crash recovery (bug :bug:`1529885`).
+
+Bugs Fixed
+==========
+
+ Clustering secondary index could not be created on a partitioned |TokuDB| table. Bug fixed :bug:`1527730` (:tokubug:`720`).
+
+ :ref:`toku_backup` was failing to compile with |Percona Server| 5.7. Bug fixed :backupbug:`123`.
+
+ When enabled, :ref:`super-read-only` could break statement-based replication while executing a multi-table update statement on a slave. Bug fixed :bug:`1441259`.
+
+ Granting privileges to user with :ref:`pam_plugin` could lead to a server crash. Bug fixed :bug:`1521474`.
+
+ |TokuDB| status variables were missing from |Percona Server| :rn:`5.7.10-1`. Bug fixed :bug:`1527364` (:tokubug:`923`).
+
+ Attempting to rotate the audit log file would result in audit log file name :file:`foo.log.%u` (literally) instead of a numeric suffix. Bug fixed :bug:`1528603`.
+
+ Adding an index to a temporary table while :variable:`expand_fast_index_creation` was enabled could lead to server assertion. Bug fixed :bug:`1529555`.
+
+ |TokuDB| would not be upgraded on *Debian*/*Ubuntu* distributions while performing an upgrade from |Percona Server| 5.6 to |Percona Server| 5.7 even if explicitly requested. Bug fixed :bug:`1533580`.
+
+ Server would assert when two different transactional engines were used within one transaction on a replication slave which has binary log enabled and disabled slave updates logging (:variable:`log_slave_update` set to ``0``). Bug fixed :bug:`1534249` (upstream bug :mysqlbug:`80053`).
```

Comment:
Server would assert when both TokuDB and InnoDB tables were used within one transaction ...
... binary log enabled and slave updates logging disabled